modules/audio_device/win/core_audio_input_win.cc

```diff
@@ -355,11 +358,29 @@ bool CoreAudioInput::OnDataCallback(uint64_t device_frequency) {
                               format_.Format.nBlockAlign * num_frames_to_read);
       RTC_DLOG(LS_WARNING) << "Captured audio is replaced by silence";
     } else {
+      // RingRTC change to shift multiple channels to stereo channels.
+      UINT16 nChannels = format_.Format.nChannels;
+
+      if (nChannels > 2) {
+        int16_t* audio_data_src = (int16_t*)audio_data;
```

Comment:
Might be more readable with something like:

for (int i = 0; i < num_frames_to_read; ++i) {
  memmove(audio_data[i * 2], audio_data[i* nChannels], 4)
}

Hopefully it isn't  slower via some compiler magic.